experimental/ir/ir_test.go

```diff
@@ -78,13 +81,17 @@ func TestIR(t *testing.T) {
 			require.NoError(t, yaml.Unmarshal([]byte(text), &test))
 		}
 
-		files := source.NewMap(maps.Collect(iterx.Map1To2(
+		var files source.Opener = source.NewMap(maps.Collect(iterx.Map1To2(
 			slices.Values(test.Files),
 			func(f File) (string, string) {
 				return f.Path, f.Text
 			},
 		)))
 
+		if !test.DisableSysroot {
```

Comment:
This feels like a strangely named flag. Why not something more directly related to the resulting behavior, like `IncludeWKTSources`?